draft-fdb-rats-psa-endorsements.md

```diff
@@ -148,41 +161,31 @@ It is RECOMMENDED to consistently provide a product identifier.
 Reference Values carry measurements and other metadata associated with the
 updatable firmware in a PSA RoT.  When appraising Evidence, the Verifier
 compares Reference Values against the values found in the Software Components
-of the PSA token (see Section 3.4.1 of {{PSA-TOKEN}}).
+of the PSA token (see {{Section 3.4.1 of PSA-TOKEN}}).
 
-Each measurement is encoded in a `measurement-map` of a CoMID
-`reference-triple-record`.  Since a `measurement-map` can encode one or more
-measurements, a single `reference-triple-record` can carry as many measurements
-as needed, provided they belong to the same PSA RoT identified in the subject of
-the "reference value" triple.  A single `reference-triple-record` MUST
-completely describe the updatable PSA RoT.
+When there is more than one measurement associated to a certain PSA RoT, the
+measurements are spread across multiple `reference-triple-record`s and, in
+certain cases, across multiple CoMIDs.  A single CoBOM MUST completely describe
+the updatable PSA RoT.
 
-The identifier of a measured software component is encoded in a `psa-swcomp-id`
-object as follows:
+The elements of the `psa-software-component` map defined in {{Section 4.4.1 of
+PSA-TOKEN}} are matched against CoMID `measurement-map` entries as follows:
 
-~~~
-{::include psa-ext/swcomp-id.cddl}
-~~~
-
-The semantics of the codepoints in the `psa-swcomp-id` map are equivalent to
-those in the `psa-software-component` map defined in Section 3.4.1 of
-{{PSA-TOKEN}}.  The `psa-swcomp-id` MUST uniquely identify a given software
-component within the PSA RoT / product.
-
-In order to support PSA Reference Value identifiers, the CoMID type
-`$measured-element-type-choice` is extended as follows:
+PSA Evidence | PSA Endorsement
```

Comment:
It would be a good and handy reference to use a third column with a short sentence what each element means

Reply:
this may not be as short as you imagine :-) adding a reference to the relevant PSA-TOKEN section instead